README.md

```
@@ -226,3 +226,4 @@ See LICENSE for more information.
 Email: bugsquashers@fairbasket.com
 
 GitHub Issues: Report a bug or request a feature
+RAG pipeline 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)